src/index.html

```diff
@@ -9,6 +9,68 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- <h1>Moyo header</h1> -->
+    <header>
+      <div class="logo">
+
+      </div>
+      <nav class="nav">
+        <ul class="nav__element">
+          <a class="nav__one nov__text is-active">
+            Apple
+          </a>
+          <div class="underline"></div>
+        </ul>
```

Comment:
`<ul>` - unordered list - so there should be 1 list, and many `<li>` - list items

Reply:
do not use separate div for underline - use pseudoelement `after` instead

Reply:
I will use <li> but for check test, me need use <a>.